src/js/components/Home/Countries.js

```diff
@@ -2,39 +2,63 @@ import React, {Component} from "react";
 import Map from "./Map";
 import './MapStyles.sass'
 import ReactFlagsSelect from 'react-flags-select';
+import Geocode from "react-geocode";
 
+// function onSelectFlag(countryCode){             //TODO: Function to be implemented
+//     console.log(countryCode)
+// }
+// set Google Maps Geocoding API for purposes of quota management. Its optional but recommended.
+Geocode.setApiKey("AIzaSyBiCz9njYP8nYxEUEQLzltCM8kAnm7z6To");
 
 class Mapbuttons extends Component {
     constructor() {
         super()
         this.state = {center: [0, 0]}
+        this.getCountryCode = this.getCountryCode.bind(this);
     }
-    changeCenter = center => () => {
-        this.setState({center})
-    }
+    // changeCenter(center) {
+    //     this.setState({center})
+    // }
 
-     onSelectFlag=countryCode=>() =>{
-        console.log(countryCode);
-        return this.changeCenter([-122.4194, 37.7749])
+    getCountryCode(country) {
+        console.log(country);
+        Geocode.fromAddress(country).then(
+            response => {
+                const { lat, lng } = response.results[0].geometry.location;
+                 this.setState({center: [lat,lng] });
+                 console.log(lat);
```

Comment:
whats the reason for pushing console logs?
if the code works, remove it. dont push logs to a production env